ckanext/datagovtheme/templates/templates_new/package/search.html

```diff
@@ -133,14 +133,18 @@ <h2 class="module-heading">
                data-module="location-autocomplete"
                data-module-source="/api/action/location_search" />
     </div>
-    <div class="dataset-map" data-module="spatial-query">
+    {% set map_config = h.get_common_map_config() %}

```

Comment:
Is this line necessary?  I think it depends on `ckanext-spatial` which I guess isn't a requirement right now.  Since it's not defined anywhere else, it's making all of the tests fail @FuhuXia

Reply:
Definition: https://github.com/ckan/ckanext-spatial/blob/992b2753fc24d0abb12ced5cf5aaa3a853ca9ea4/ckanext/spatial/helpers.py#L66

Reply:
Oh, but it was trying to execute the function anyway.  But okay.